micronaut-camunda-bpm-example/src/main/resources/META-INF/services/org.camunda.bpm.engine.rest.spi.ProcessEngineProvider

```diff
@@ -0,0 +1 @@
+info.novatec.micronaut.camunda.bpm.example.rest.spi.ServletProcessEngineProvider
```

Comment:
must the provider be created like this? Cannot we just register it as a bean? If not: why like this?